modules/virt-example-host-vrf.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+[id="virt-example-host-vrf_{context}"]
+= Example: Network interface with a VRF instance node network configuration policy
+
+Associate a Virtual Routing and Forwarding (VRF) instance with a network interface by applying a `NodeNetworkConfigurationPolicy` custom resource (CR). 
+
+:FeatureName: Associating a VRF instance with a network interface
+include::snippets/technology-preview.adoc[]
+
+By associating a VRF instance with a network interface, you can support traffic isolation, independent routing decisions and the logical separation of network resources. 
+
+In a bare-metal environment, you can announce load balancer services through interfaces belonging to a VRF instance by using MetalLB. For more information, see the _Additional resources_ section.
```

Comment:
@adellape I don't think we have good guidance on when to use quotes vs italics when referencing other sections/modules/assemblies, etc. I think we're all over the place now and it would probably be a good idea to decide one way and get it into the guidelines.